### ISSUE TITLE
Colloquy goes into a state where one can't type anything

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#1415]
*Issue migrated from trac ticket [#1190](http://colloquy.info/project/ticket/1190)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Blocker | **priority:** Highest | **status:** closed | **resolution:** Works for Me
___
#### 2008-01-31 19:29:34: shreevatsa created the issue

I often get into a situation where I can't type anything in Colloquy windows. I am not sure how exactly to reproduce it, but here is something that seems to mostly work:

 - Type something (in some channel, not in the server console or other windows)
 - Hit Cmd-A (for Select All)
 - Hit Option-Delete (Backspace) [maybe Cmd-Delete also works here?]
 - Hit Cmd-Z (for Undo)

Now, if everything goes (not) well, you can't type anything at all in any of the chat windows or the server console. You *can* paste from other windows. You *cannot* hit Return to send the pasted message. You *can* hit Cmd-J (for example) and type channel names in the window that pops up. You just can't type in any chat windows.

Minimising and restoring Colloquy does not help; the only way out is to restart Colloquy.
___
#### 2008-02-07 15:45:51: shreevatsa commented

Today I deleted something and hit Cmd-Z, and Colloquy crashed.

___
#### 2008-02-09 21:25:31: shreevatsa commented

 - The Cmd-A seems unnecessary, even deleting something simply with Option-Delete (and then hitting Cmd-Z) seems to screw it up sometimes.

 - Option-Delete seems unnecessary too; deleting something with just Delete (and then hitting Cmd-Z) seems to screw it up too.

Can I turn on debugging or something to see what's happening? It is not perfectly reproducible but happens fairly frequently.

___
#### 2008-02-10 06:12:43: shreevatsa commented

I can also reproduce this by using Ctrl-K and then Cmd-Z.

Basically, clearing text (in whatever way) and doing Undo causes this annoying bug. I wish I could help debug it, but I don't know how to...

___
#### 2008-12-15 10:15:10: @janeylicious changed status from *new* to *closed*

___
#### 2008-12-15 10:15:10: @janeylicious changed resolution from ** to *worksforme*

___
#### 2008-12-15 10:15:10: @janeylicious commented

please reopen if you continue to have problems (including more details about which colloquy version you're using, your OS, etc.). can't reproduce in latest colloquy on 10.5.5.
